### DISCUSSION TITLE
Skin editor crash when debugging.

### DISCUSSION THREAD

**Discussion by deok420** [ppy/osu#20971] [Q&A] [answered]
Note: This only happens when I use the "Debug" project configuration.
I'm making a new skin editor for lazer, because the current one is very plain, but I keep getting this error when debugging. If I press continue it would just keep putting it up again. 
Here are the error details: https://pastebin.com/EP8KsckX

**Comment by peppy** [comment, marked as answer]
Please run the release build. I'm not sure why this is occurring for you, but I'd say either your IDE is presenting caught exceptions, or you have local code changes breaking this flow.
